support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.tsx

```diff
@@ -368,7 +368,7 @@ function WeeklyCheckoutForm(props: PropTypes) {
 					</FormSection>
 					{!props.billingAddressMatchesDelivery ? (
 						<FormSection title="Your billing address">
-							<BillingAddress countries={billableCountries} />
+							<BillingAddress countries={gwCountries} />
```

Comment:
Presume its ok to pay from a Guardian Weekly Non-Deliverable country just not deliver there?

Reply:
I've asked this question to the stakeholders. I've also updated list of countries where users can buy from.

Reply:
Confirmed, it is OK to buy from a country we don't deliver to, for a paper to be delivered to a country where we do.